lib/Conversion/TritonGPUToLLVM/ScanOpToLLVM.h

```diff
@@ -0,0 +1,15 @@
+#ifndef TRITON_CONVERSION_TRITONGPU_TO_LLVM_SCAN_OP_H
```

Comment:
nit: use `#pragma once`

Reply:
This doesn't seem to be the convention followed in triton project.